controllers/operands/kubevirt.go

```diff
@@ -509,10 +521,30 @@ func getKVConfig(hc *hcov1beta1.HyperConverged) (*kubevirtcorev1.KubeVirtConfigu
 	if hc.Spec.ResourceRequirements != nil {
 		config.AutoCPULimitNamespaceLabelSelector = hc.Spec.ResourceRequirements.AutoCPULimitNamespaceLabelSelector.DeepCopy()
 	}
-
 	return config, nil
 }
 
+func getNetworkBindings(
+	hcoNetworkBindings map[string]kubevirtcorev1.InterfaceBindingPlugin,
+	hcoFeatureGates hcov1beta1.HyperConvergedFeatureGates) (map[string]kubevirtcorev1.InterfaceBindingPlugin, error) {
+	networkBindings := maps.Clone(hcoNetworkBindings)
+
+	if hcoFeatureGates.PrimaryUserDefinedNetworkBinding != nil && *hcoFeatureGates.PrimaryUserDefinedNetworkBinding {
+		if networkBindings == nil {
+			networkBindings = make(map[string]kubevirtcorev1.InterfaceBindingPlugin)
+		}
+
+		var sidecarImage string
+		var ok bool
+		if sidecarImage, ok = os.LookupEnv(hcoutil.PrimaryUDNImageEnvV); !ok {
+			return nil, errors.New("failed to get primary UDN image env Var")
```

Comment:
Not sure about this error. This is not something the user can fix. Maybe it will better to check it on boot time and exit, to fail the pod? @tiraboschi WDYT?

Reply:
I agree: this is something we are configuring at the bundle build time as part of the opinionated deployment and it's not something the user can or should try to fix.